src/algorithms/calorimetry/CalorimeterHitDigiConfig.h

```diff
@@ -0,0 +1,34 @@
+// Subject to the terms in the LICENSE file found in the top-level directory.
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace eicrecon {
+
+  struct CalorimeterHitDigiConfig {
+
+    std::vector<double>      eRes;
+    double                   tRes;
+
+    // single hit energy deposition threshold
+    double                   threshold{0};
+
+    // digitization settings
+    unsigned int             capADC{1};
+    double                   capTime{1}; // FIXME unsigned int?
```

Comment:
This is confusing, #795 
```suggestion
    double                   capTime{1000}; // dynamic range in ns
```
Fixes test since it removed the test hits at 9 ns for being out of time window.